docs/MO_DG/prepare_model/Config_Model_Optimizer.md

```diff
@@ -453,15 +453,11 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
 ```shell
 pip3 install -r requirements.txt
 ```
-> **NOTE**: TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
+> **NOTE**: Support of MO in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models. Use separate virtual environments if you want to install multiple TensorFlow versions.
 
     *   To install dependencies only for Caffe:
 ```shell
 pip3 install -r requirements_caffe.txt
-```
-    *   To install dependencies only for TensorFlow 1.x:
-```shell
-pip3 install -r requirements_tf.txt
 ```
     *   To install dependencies only for TensorFlow 2.x:
```

Comment:
To install dependencies only for TensorFlow

Reply:
and please correct `_tf2` -> `_tf`

Reply:
Done.